src/Chapter05.Tests/Listing05.32.Tests.cs

```
@@ -5,5 +5,22 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter05.Listing05_32.Tests;
 [TestClass]
 public class ProgramTests
 {
+    [TestMethod]
+    public void Main_NoArgs_ExpectErrors()
+    {
+        string expected = "Value cannot be null. (Parameter \'httpsUrl\')";
 
+        IntelliTect.TestTools.Console.ConsoleAssert.Expect(expected,
+        () =>
+        {
+            try
+            {
+                Program.Main();
+            }
+            catch (Exception ex)
```

Same comment here, we don't want to suppress the failure cases.